.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,43 @@
 <!-- Please review the following before submitting a PR:
 Che's Contributing Guide: https://github.com/eclipse/che/blob/master/CONTRIBUTING.md
 Pull Request Policy: https://github.com/eclipse/che/wiki/Development-Workflow#pull-requests
-
 -->
 
 ### What does this PR do?
 
 
+### Screenshot/screencast of this PR
+<!-- Please include a screenshot or a screencast explaining what is doing this PR -->
```

Comment:
It all looks OK to me I'm just wondering if we need a screenshot/screencast for things in this repo

Reply:
it's optional but it could be what you bring with a devfile

For example if I add a quarkus devfile, then I can see a screenshot of theia + project + quarkus being started 

or for a petclinic app, a screenshot of the running app